### PR TITLE
Publish technical documentation to the website repository

### PR DESCRIPTION
**Pull request by jdbaldry**
Documentation can be viewed locally by running `make docs` from the `docs` directory.

I've modified the Docusaurus admonitions to be plain GitHub Flavored Markdown blockquotes so that the same Markdown files can be used in both sites.

I've also configured Docusaurus to hide the titles in those pages so that we can manually set the first heading like we do in Hugo.

The `publish-technical-documentation-next` workflow publishes the content to the website repository on every push to the `main` branch. https://github.com/grafana/website/pull/13617 in the website repository will be used to mount the content into the appropriate place in the grafana.com site.

### CODE REVIEW

File: docs/sources/_index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Grafana Scenes
+---
+# Grafana Scenes
+
+**`@grafana/scenes` is currently in _public preview_.**
+
+Build highly interactive Grafana apps with ease.
```
Comment (imatwawana) [inline docs/sources/_index.md:8]:
```suggestion
Grafana Scenes is a simple and intuitive API that allows you quickly build experiences similar to Grafana Dashboards. Grafana Dashboards features, like querying and transformations, templating, time ranges, and more, are available in Grafana Scenes. Scenes is built with extensibility in mind so you can build your own Scene objects to provide custom behaviors.

If you want to play with Scenes right away, use [@grafana/create-plugin](https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/README.md).

To create a Scenes app, run the following command:
```
npx @grafana/create-plugin@latest
```

The following topics provide more detailed information for using Scenes:
```

File: docs/sources/_index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Grafana Scenes
+---
+# Grafana Scenes
+
+**`@grafana/scenes` is currently in _public preview_.**
```
Comment (imatwawana) [inline docs/sources/_index.md:6]:
```suggestion
{{% admonition type="note" %}}
Scenes is currently in public preview.
{{% /admonition %}}
```

**Review by imatwawana — COMMENTED** [review]
The information architecture needs to be applied to the docs. I've made some updates to the Scenes index file to flesh it out a bit.

**Comment by torkelo** [comment]
Hm.. do we sync plugin-tools docs to website? 
https://grafana.github.io/plugin-tools/

**Comment by jdbaldry** [comment]
> Hm.. do we sync plugin-tools docs to website? https://grafana.github.io/plugin-tools/

I believe we do not but intend to discuss that with the team responsible.

**Comment by jdbaldry** [comment]
> The information architecture needs to be applied to the docs. I've made some updates to the Scenes index file to flesh it out a bit.

Information architecture should be sorted now. I still need to work on the images.

**Comment by GrafanaWriter** [comment]
> Hm.. do we sync plugin-tools docs to website?
> https://grafana.github.io/plugin-tools/
We don't sync the site, per se, since the content is also published to grafana.com/docs/grafana/latest/developers/](https://grafana.com/docs/grafana/latest/developers/plugins/ - @josmperez is working with @tolzhabayev and team on harmonizing those docs.

The long term plan, @torkelo, is to build out a developer portal with Scenes and contributor plugins docs as well as other content (TBD). The goal of this PR is to ensure the Scenes docs are published in time for G10 launch using the available docs website.

**Comment by imatwawana** [comment]
@GrafanaWriter @jdbaldry The IA still isn't quite right and I'm concerned that with @dprokop out tomorrow (and already gone for the day), we don't have sufficient time to properly review this PR for Monday G10 launch. This seems to be introducing unnecessary risk given that the Scenes website is already referenced in the What's new.

**Comment by jdbaldry** [comment]
Images and IA should now match the existing site.

File: docs/sources/scene-objects/scene-app/scene-app-drilldown.md [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+
+# Drill-down pages in Scenes apps
+
+Drill-down pages are a powerful tool for building complex, data-driven applications. They allow you to create a high-level overview of the data that a user can interact with and progressively explore to reveal underlying data.
+
+## Add drill-down pages to Scenes apps
+
+`SceneAppPage` comes with an API that allows you to create deep, nested drill-down pages.
+
+> **Note:** **Before you begin**: You must already know about React Router URL params, Grafana field configuration, and data links before continuing with this guide.
```
Comment (imatwawana) [inline docs/sources/scene-objects/scene-app/scene-app-drilldown.md:17]:
```suggestion
> **Note:** Before you begin, you must already know about React Router URL params, Grafana field configuration, and data links before continuing with this guide.
```

File: docs/sources/scene-objects/scene-app/scene-app-drilldown.md [hunk 4]
```diff
@@ -0,0 +1,381 @@
...
+        getScene: getOverviewScene,
+      }),
+    ],
+  });
+}
+```
+
+### Step 3. Set up drill-down navigation
+
+To show the drill-down page, you need to provide navigation. Configure Table panel data links (learn about data links in the [official Grafana documentation](https://grafana.com/docs/grafana/latest/panels-visualizations/configure-data-links/)). Then modify the Table panel configuration to set up a data link for the `handler` field:
```
Comment (imatwawana) [inline docs/sources/scene-objects/scene-app/scene-app-drilldown.md:81]:
```suggestion
To show the drill-down page, you need to provide navigation. First, configure Table panel [data links](https://grafana.com/docs/grafana/latest/panels-visualizations/configure-data-links/), then modify the Table panel configuration to set up a data link for the `handler` field:
```

File: docs/sources/advanced-usage/_index.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
+title: Advanced Usage
+weight: 300
+---
+# Advanced Usage
+
```
Comment (imatwawana) [inline docs/sources/advanced-usage/_index.md:6]:
```suggestion

Do more with Scenes with custom scene objects, activation handlers, and variables, data, and time range in custom scene objects. 

```

File: docs/sources/advanced-usage/_index.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+title: Advanced Usage
+weight: 300
```
Comment (imatwawana) [inline docs/sources/advanced-usage/_index.md:3]:
```suggestion
weight: 300
description: "Do more with Scenes with custom scene objects and activation handlers."
```

**Comment by GrafanaWriter** [comment]
> @GrafanaWriter @jdbaldry The IA still isn't quite right and I'm concerned that with @dprokop out tomorrow (and already gone for the day), we don't have sufficient time to properly review this PR for Monday G10 launch. This seems to be introducing unnecessary risk given that the Scenes website is already referenced in the What's new.

We need to publish via the grafana.com/docs site until we have a developer portal.

File: docs/sources/advanced-usage/advanced-activation-handlers.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
+---
+id: advanced-activation-handlers
+title: Activation handlers
+hide_title: true
+weight: 200
+---
+
+# Activiation handlers
```
Comment (imatwawana) [inline docs/sources/advanced-usage/advanced-activation-handlers.md:8]:
```suggestion
# Activation handlers
```

**Review by imatwawana — COMMENTED** [review]
The links internal to the doc set aren't working, I assume, because they're still formatted for the docusaurus site. Do these just need to be relrefs? Also the links that used to be external that were pointing to Grafana docs are full URLs, so I assume those need to change as well. 

I've made several edits to try to create parity with the original published docs and added some intro text on the level-2 pages (the way docusaurus does the sidebar, those aren't real pages so they didn't need any content before). I also changed some of the link text where the links were pointing to Grafana docs before. I also added descriptions in front matter where those were missing before.

File: docs/sources/advanced-usage/advanced-activation-handlers.md (hunk 7 above)
Comment (GrafanaWriter) [reply docs/sources/advanced-usage/advanced-activation-handlers.md:8]: Thanks for making these changes, @imatwawana. @jdbaldry - These look good to me. The only open question is the one related to linking - can the doc-validator fix those?

**Review by torkelo — COMMENTED** [review]
@GrafanaWriter 
> We don't sync the site, per se, since the content is also published to grafana.com/docs/grafana/latest/developers/](https://grafana.com/docs/grafana/latest/developers/plugins/ - @josmperez is working with @tolzhabayev and team on harmonizing those docs.

I don't see the content on https://grafana.github.io/plugin-tools/docs available on https://grafana.com/docs/grafana/latest/developers/plugins/   but maybe not looking in the right spot.

> We need to publish via the grafana.com/docs site until we have a developer portal.

why? Can't we wait for it?. This PR seems to duplicate all the content making it a lot harder to maintain. But maybe those copies are built by script and we do not need to check them into source control?

**Comment by jdbaldry** [comment]
I'm only duplicating at the moment because I wasn't sure if we needed to maintain both. I should have asked already haha.

It makes my life a lot easier to ditch the docusaurus part so that I don't have to maintain the translations.

@GrafanaWriter, @torkelo 
Do we need to get together to agree an approach before GrafanaCon because I would rather not invest more time in this PR if it's not going to be merged.

File: .github/workflows/publish-technical-documentation-next.yml [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+          repository: "grafana/website"
+          branch: "publish-scenes-to-website"
+          host: "github.com"
+          # PUBLISH_TO_WEBSITE_TOKEN is a fine-grained GitHub Personal Access Token that expires.
+          # It must be regenerated in the grafanabot GitHub account and requires a Grafana organization
+          # GitHub administrator to update the organization secret.
+          # The IT helpdesk can update the organization secret.
+          github_pat: "grafanabot:${{ secrets.PUBLISH_TO_WEBSITE_TOKEN }}"
+          source_folder: "docs/sources"
+          target_folder: "content/docs/scenes/next"
```
[the remark this answers is not in the document]
Comment (jdbaldry) [reply .github/workflows/publish-technical-documentation-next.yml:39]: We do a little magic on the website side where Hugo mounts this content into the correct location. Hugo has a pre-build virtual filesystem and we mount `content/docs/scenes/next` into `content/docs/grafana/next/developers/scenes`.

The patch to do that is in https://github.com/grafana/website/pull/13617/files.

**Comment by torkelo** [comment]
@jdbaldry @GrafanaWriter yes, that would be good. I was not aware that we had plans to add these docs to the main docs site, I thought using a separate site like plugin-tools docusaurus site was good enough for now until we better more unified developer portal solution.

**Comment by jdbaldry** [comment]
No longer needed.

**Pull request closed without merging**